modules/ROOT/pages/fd-connection-delete.adoc

```diff
@@ -0,0 +1,36 @@
+= Delete a Connection
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can delete a connection that is being used by or is available to be used by a specific type of connector. For example, if the connection is available for the Database connector.
+
+To delete a connection, you must log into Anypoint Platform with the user ID that was used to create the connection. You cannot delete a connection that was created under a user ID that you are not logged in with.
+
+If the connection is shared because you selected the *Share this connection with my business group* checkbox when you created it, deleting the connection affect all of the connectors of that type in all of the Mule-app projects and running Mule apps in your business group.
+
+If a Mule-app project is using the connection, an error indicator appears on the card that represents the connector. The error indicator appears when a configuration that uses the connection is reopened, the project is reopened, a user tries to add a new configuration for the same connector type, or a user clicks *Test* in the project.
+
+If a deployed Mule app is using the connection, the app continues to do so. If you modify the Mule app, you must switch to a different connection before you redeploy it.
+
+*To delete a connection:*
```

Comment:
Avoid using these inline headings, per previous edit comments.
Try to separate the conceptual information from the task steps.
GLOBALs